src/commands/cluster-getkeysinslot.json

```diff
@@ -1,7 +1,7 @@
 {
     "GETKEYSINSLOT": {
         "summary": "Return local key names in the specified hash slot",
-        "complexity": "O(log(N)) where N is the number of requested keys",
+        "complexity": "O(N) where N is the number of requested keys",
```

Comment:
i didn't look at the code, but digging in the blame log i found this: https://github.com/redis/redis-doc/commit/63f8c6803b6b2e615bc42b51451255ca5ee1b26c

Reply:
well, the code just runs on a linked list today.. 
could be somehow related to the fact it used to be a rax / skiplist in the past..

Reply:
yean